web-admin-dashboard/src/questionrepo/question.model.ts

```diff
@@ -1,10 +1,10 @@
 export interface Question {
-  qId: number;
-  title: string;
-  category: string[];
-  complexity: QuestionDifficulty;
-  link: string;
-  description: string;
+  _id: string;
+  questionId: number;
+  questionTitle: string;
+  questionCategories: string[];
+  questionDifficulty: QuestionDifficulty;
+  questionDescription: string;
```

Comment:
Minor: Not too familiar with what you all decided on for naming conventions, but why are there two id fields? And would be good to remove the `question` prefix from the other fields since the object is already called `Question`

Then it can look like `question.title` instead of `question.questionTitle`.